README.md

```diff
@@ -1,9 +1,66 @@
-# triton-docker
+# Triton Docker CLI helper
 
-Docker CLI helper for Triton
+This script installs known-good, tested versions of the Docker (now Moby) and Docker Compose CLI tools for use with Triton.
+
+Additionally, this script will automatically configure those tools for use with Triton when you use them, making it easy to switch between Docker on your laptop and Docker on Triton.
+
+### Compatibility and requirements
+
+This is designed and tested for Linux and MacOS X.
+
+This code also requires [the Triton CLI tools](https://docs.joyent.com/public-cloud/api-access/cloudapi) and [a Triton account](https://docs.joyent.com/public-cloud/getting-started) on either the Triton Public Cloud (Joyent public cloud) or in a private cloud powered by Triton.
```

Comment:
"Triton Public Cloud" is not the product name. It should be called Triton Compute Services or Triton public cloud.

I find myself being a stickler about this due to the many conversations we had about this around the new site launch. If this has changed, please let me know.